app/Http/Controllers/SupplyController.php

```diff
@@ -144,7 +145,6 @@ public function logNetworkClick(Request $request, $id)
 
         $logIp = bin2hex(inet_pton($request->getClientIp()));
 
-
         $cid = Utils::getRawTrackingId($request->query->get('cid'));
         $tid = Utils::getRawTrackingId($request->cookies->get('tid')) ?: $logIp;
         // TODO get user / website / zone
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)